packages/editor/src/store/reducer.js

```diff
@@ -155,6 +156,13 @@ export const editor = flow( [
 
 		return state;
 	} ),
+	selection( state = { start: {}, end: {} }, { type, selection } ) {
```

Comment:
I'm probably missing something, but I'm not understanding why we need to keep selection state in the editor.

Reply:
Otherwise it's not tracked in `withHistory`. Otherwise `withHistory` only contains the content, not the selection.